src/features/game/types/pets.ts

```diff
@@ -554,7 +554,7 @@ export function getPetRequestXP(food: CookableName) {
   return PET_REQUEST_XP[difficulty];
 }
 
-const MAX_PET_LEVEL = 200;
```

Comment:
[[PROPOSAL] Pets](<https://github.com/sunflower-land/sunflower-land/discussions/5657>)
> _There are no level caps _

This comment and a previous comment about L100 Warthog (or something like that) was in the proposal.

Was the design changed to a level cap of 200?